app/controllers/system/admin/instances_controller.rb

```diff
@@ -4,7 +4,10 @@ class System::Admin::InstancesController < System::Admin::Controller
   add_breadcrumb :index, :admin_instances_path
 
   def index #:nodoc:
-    @instances = Instance.order_by_id.page(page_param).calculated(:course_count, :user_count)
+    # Put the default instance at the start of the collection
+    instances = [Instance.default.calculated(:course_count, :user_count)] +
+                Instance.without_default.order_by_name.calculated(:course_count, :user_count)
+    @instances = Kaminari.paginate_array(instances).page(page_param)
```

Comment:
Any ways to directly fetch all instances from DB and order the default one first?  This way will force all instances to be loaded, even it's not displayed.